### PR TITLE
Tags feature

### PR DESCRIPTION
**Pull request by brunopenso**
PR with tags feature implemented. Just use @something and the end of the string.

When listing the times, timet will show the minutes/hours spend by tags too.

I tried to use the #something but node understand that this is a comment.

### CODE REVIEW

File: lib/list.js [hunk 1]
```diff
@@ -30,10 +30,28 @@ module.exports = function (filter, callback) {
     db.find({date: Number(date.format('YYYYMMDD'))}).sort({ datetime: 1 }).exec(function (err, registers) {
       if (err) return callback(err)
       var totalTime = 0
+      var totalTimeObj = {}
+      var tags = []
       for (var i in registers) {
-        totalTime += registers[i].time
+        var itemTime = registers[i].time
+        totalTime += itemTime
+        registers[i].tags.map(item => {
```
Comment (fabiorogeriosj) [inline lib/list.js]: Please check this, I think is out `undefined`

File: lib/list.js [hunk 2]
```diff
@@ -30,10 +30,28 @@ module.exports = function (filter, callback) {
...
+          }
+        })
       }
+
+      var timeByTag = ''
+      tags.map(item => {
+        timeByTag += item + ' ' + checkShowTotal(totalTimeObj[item]) + ' | '
+      })
+
       console.log(' ', (' DATE: ' + date.format(config.format) + ' ').bgWhite.black, checkShowTotal(totalTime).bgGreen.black)
```
Comment (fabiorogeriosj) [inline lib/list.js]: Add timeByTag on the same line

File: lib/list.js (hunk 1 above)
Comment (brunopenso) [reply lib/list.js]: Fixed

File: lib/list.js (hunk 2 above)
Comment (brunopenso) [reply lib/list.js]: Done

**Comment by brunopenso** [comment]
@fabiorogeriosj adjustments done

**Pull request merged**